resources/views/api/master.blade.php

```diff
@@ -53,6 +43,18 @@
         <!-- footer -->
         <div>
             @include("api.includes.sections.footer")
-        </div>    
+        </div> 
+
+        <!-- <script src = "{!! load_asset('/js/api.js') !!}"></script>
+        <script src = "{!! load_asset('/js/api.js') !!}"></script>
+        <script src="{!! load_asset('/js/audio.min.js') !!}"></script>
+        <script src="{!! load_asset('/js/like.js') !!}"></script>
+        <script src = "{!! load_asset('/js/api.main.js') !!}"></script>
+        <script src="{!! load_asset('/js/audio.min.js') !!}"></script>
+        <script src = "{!! load_asset('/js/comment.js') !!}"></script> -->
+        <script src = "{!! load_asset('/js/scripts.js') !!}"></script>
+
+       <!--  <script src="{!! load_asset('/js/socialmediashare.js') !!}"></script>
+        <script src="{!! load_asset('/js/commentupdate.js') !!}"></script>    -->
```

Comment:
**Code Quality - Nitpick**

Please remove unused and unnecessary comments here